site/content/docs/main/resource-filtering.md

```diff
@@ -359,4 +359,103 @@ Velero supported conditions and format listed below:
 
 **Resource policies rules**
 - Velero already has lots of include or exclude filters. the resource policies are the final filters after others include or exclude filters in one backup processing workflow. So if use a defined similar filter like the opt-in approach to backup one pod volume but skip backup of the same pod volume in resource policies, as resource policies are the final filters that are applied, the volume will not be backed up.
-- If volume resource policies conflict with themselves the first matched policy will be respected when many policies are defined.
\ No newline at end of file
+- If volume resource policies conflict with themselves the first matched policy will be respected when many policies are defined.
+
+
+**Support for `fs-backup` and `snapshot` actions via volume policy feature**
```

Comment:
Let's clarify two more things in the doc:
1. The behavior of volumePolicy working together with `--snapshot-volumes`
2. The behavior of snapshot data mover --- data mover should go with the existing options (-`-snapshot-move-data` and `--default-snapshot-move-data`, but the volumePolicy for `snapshot` also affects data mover)

Reply:
Done, updated the PR